pallets/phala/src/mining.rs

```diff
@@ -465,8 +476,8 @@ pub mod pallet {
 			let budget_per_block = FixedPoint::from_bits(tokenomic.budget_per_block);
 			let new_budget = budget_per_block * fp!(0.75);
 			tokenomic.budget_per_block = new_budget.to_bits();
-			Self::update_tokenomic_parameters(tokenomic);
 			Self::deposit_event(Event::<T>::SubsidyBudgetHalved);
+			Self::update_tokenomic_parameters(tokenomic);
```

Comment:
Why the order matters?

Reply:
Because otherwise it shows "Update" earlier than "Halving". Actually the halving happens before the parameters are updated.